### PR TITLE
Edited for grammar and clarity

### PR DESCRIPTION
**Pull request by schindld**
Made small spelling and grammar changes to entire document (up to tip 20).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ person.sayName().changeName("Timmy").sayName();
 
 > 2016-01-19 by [@gogainda](https://twitter.com/gogainda)
 
-Suppose you have a couple of variables with unknown types and you want to concatenate them. To be sure that the arithmetical operation would not be applied during the concatenation:
+Suppose you have a couple of variables with unknown types and you want to concatenate them in a string. To be sure that the arithmetical operation is not be applied during concatenation, use ```concat```:
```
Comment (homer0) [inline README.md]: To write inline code (without language), you can use only one **`**

File: README.md [hunk 2]
```diff
@@ -835,19 +835,19 @@ In this case the `$digest` method starts the `$digest` cycle for the current sco
...
 
 
 ## #0 - Insert item inside an Array
 > 2015-12-29
 
-Inserting an item into an existing array is a daily common task. You can add elements to the end of an array using push, to the beginning using unshift, or the middle using splice.
+Inserting an item into an existing array is a daily common task. You can add elements to the end of an array using push, to the beginning using unshift, or to the middle using splice.
 
-But those are known methods, doesn't mean there isn't a more performant way, here we go...
+But those are known methods... doesn't mean there isn't a more performant way. Here we go:
```
Comment (homer0) [inline README.md]: Shouldn't you add a `but it/which` before the `doesn't`? (honest question :P)

**Comment by homer0** [comment]
Besides those two small comments: :skull: like it!

File: README.md (hunk 1 above)
Comment (schindld) [inline README.md]: I just copied that from further down (lines 68 and 70). I wasn't trying to fix code, just grammar. But I can clean the back-ticks up in a new commit.

File: README.md (hunk 2 above)
Comment (schindld) [inline README.md]: Good suggestion. I'll update it.

**Comment by gogainda** [comment]
:100:

**Comment by loverajoel** [comment]
Thanks!

**Pull request merged**